docs/deprecated.md

```diff
@@ -558,12 +558,12 @@ backported), there is no reason to keep maintaining the `overlay` storage driver
 ### Device mapper storage driver
 
 **Deprecated in Release: v18.09**
-**Disabled by default in Release: v23.0**
```

Comment:
Probably this line can be safely retained

Reply:
Works for me; I can add it back